pkg/build/config/version.go

```diff
@@ -121,7 +121,7 @@ func CheckDroneTargetBranch() (VersionMode, error) {
 	if rePRCheckBranch.MatchString(target) {
 		return PullRequestMode, nil
 	}
-	fmt.Printf("unrecognized target branch: %s, defaulting to %s\n", target, PullRequestMode)
+	fmt.Printf("unrecognized target branch: %s, defaulting to %s", target, PullRequestMode)
```

Comment:
The newline was added on purpose to not break `go test` output, see: https://github.com/grafana/grafana/pull/102795

Is there any reason to remove it?

Reply:
I'll add this back.